src/api/app/controllers/webui/kiwi/images_controller.rb

```diff
@@ -43,16 +43,20 @@ def update
         end
         redirect_to action: :show
       rescue ActiveRecord::RecordInvalid, Timeout::Error => e
-        flash[:error] = "Cannot update repositories for kiwi image: #{@image.errors.full_messages.to_sentence} #{e.message}"
+        flash[:error] = "Cannot update kiwi image: #{@image.errors.full_messages.to_sentence} #{e.message}"
         redirect_back(fallback_location: root_path)
       end
 
       private
 
       def image_params
         params.require(:kiwi_image).permit(repositories_attributes:
```

Comment:
It is written like that because there is a bug in Rubocop, I'll report it and maybe disable the cop here.